src/inputs/AudioInput.js

```diff
@@ -8,13 +8,16 @@ export default (store) => {
     let bands
     window.setInterval(() => {
       let state = store.getState()
+
       input.normalizeLevels = state.nodes['audioNormalizeLevels'].value
       input.levelsFalloff = Math.pow(state.nodes['audioLevelsFalloff'].value, 2)
       input.maxLevelFalloffMultiplier = 1 - Math.pow(state.nodes['audioNormalizeRangeFalloff'].value, 3) * 0.01
       input.smoothing = state.nodes['audioLevelsSmoothing'].value * 0.99
       input.levelsPower = state.nodes['audioLevelsPower'].value * 3 + 0.5
-      bands = input.update()
+
+      bands = input.update(state.settings)
       store.dispatch(inputFired('audio', bands, { type: 'audio' }))
+      store.dispatch(inputFired('texture', input.texture, { type: 'texture' }))
```

Comment:
need to get this to be assigned to `input.texture.audioTexture.value` with texture being a dictionary rather than the texture object. correct solution being to make a `texturedFired` function?